src/tabs.scss

```diff
@@ -124,24 +125,25 @@
   &.--dark {
     .react-tabs {
       background-color: $rpf-grey-800;
```

Comment:
Do we need to change this value to rpf-grey-600 too so that the tab won't change colour when selected?  From the notes in the ticket it looks like "Selected" and "Unselected" tabs should be the same colour